lua/hlchunk/mods/indent/init.lua

```diff
@@ -191,8 +191,7 @@ function IndentMod:createAutocmd()
     end)
 
     local autocommands = {
-        { events = { "User" }, pattern = "WinScrolledX", opts = { lazy = false } },
-        { events = { "User" }, pattern = "WinScrolledY", opts = { lazy = true } },
+        { events = { "WinScrolled" }, opts = { lazy = true } },
```

Comment:
Not sure about the value for `lazy = true` here: in my testing both seemed to work fine, so I chose the option that sounded more performant :grinning: